tests/unit/cli/cli-test.js

```diff
@@ -5,6 +5,8 @@ var MockUI        = require('console-ui/mock');
 var MockAnalytics = require('../../helpers/mock-analytics');
 var CLI           = require('../../../lib/cli/cli');
 var td = require('testdouble');
+var heimdall = require('heimdalljs');
+var chalk = require('chalk');
```

Comment:
holdover from a test that got moved to instrumentation-test.

i'll remove it here